### PR TITLE
Adtira/redline

### PR DESCRIPTION
**Pull request by tirada**
- Polish ListView Styling
- Removed unnecessary CSS
- Applied Flex where appropriate
- Added HTML stripping to text display
- Fixed SettingsView spacing
- Fixed truncation in ListView
- Edited Task Module Parameters

### CODE REVIEW

File: src/css/App.css [hunk 1]
```diff
@@ -39,45 +39,11 @@
 }
 
 .listItemImage {
-  margin-right: 12px;
   background-color: #abcdef;
```
Comment (FaizChishtie) [inline src/css/App.css]: rip app.css

File: src/App.tsx [hunk 2]
```diff
@@ -21,11 +21,10 @@ export const App: React.FC<IAppProps> = (props: IAppProps): JSX.Element => {
     );
   } else {
     return (
-      <div>
+      <div style={{ backgroundColor: '#F3F2F1' }}>
```
Comment (sachingupta) [inline src/App.tsx]: do not use hardcoded color lets use startdust color variable so that we get right color in each theme

File: src/App.tsx [hunk 3]
```diff
@@ -4,11 +4,20 @@ import './css/App.css';
 import { getFrameContext } from './utils/utils';
 import { SettingsView } from './components/SettingsView';
 import { ContentView } from './components/ContentView';
+import { createComponent } from '@stardust-ui/react';
 
 interface IAppProps {
   onThemeChange: (theme: string) => void;
 }
 
+const ContentViewWrapper = createComponent({
```
Comment (sachingupta) [inline src/App.tsx]: move to a contentView file

File: src/App.tsx [hunk 4]
```diff
@@ -4,11 +4,20 @@ import './css/App.css';
...
+  displayName: 'ContentViewWrapper',
+  render: ({ stardust, onThemeChange }) => {
+    const { classes } = stardust;
+    return <ContentView className={classes.root} onThemeChange={onThemeChange} />;
```
Comment (sachingupta) [inline src/App.tsx]: rename to customClass?

File: src/components/ListView.tsx [hunk 5]
```diff
@@ -20,29 +19,27 @@ export const ListView: React.FC<IItemListProps> = (props: IItemListProps): JSX.E
...
+            <Flex.Item grow size="size.half">
+              <Text truncated size="medium" weight="regular" content={stripHTML(item.preview.subTitle)} />
             </Flex.Item>
           ) : null}
+          <Flex.Item>
+            <Icon name="more" />
+          </Flex.Item>
         </Flex>
       ),
-      className: 'listItem',
```
Comment (sachingupta) [inline src/components/ListView.tsx]: pass margin here.

File: src/components/RadioIcons.tsx [hunk 6]
```diff
@@ -17,26 +17,26 @@ export const RadioIcons: React.FC<IRadioIconsProps> = (props: IRadioIconsProps):
...
-        styles={styles}
-      />
-    </div>
+    <Flex styles={props.styles} vAlign="center">
+      <Flex.Item>
+        <Button
+          icon="menu"
+          iconOnly
+          text
+          styles={{ color: 'black' }}
```
Comment (sachingupta) [inline src/components/RadioIcons.tsx]: use stradust color var.

File: src/components/RadioIcons.tsx [hunk 7]
```diff
@@ -17,26 +17,26 @@ export const RadioIcons: React.FC<IRadioIconsProps> = (props: IRadioIconsProps):
...
+          onClick={(e: React.SyntheticEvent): void => handleChange(e, { value: viewType.List })}
+        />
+      </Flex.Item>
+      <Flex.Item>
+        <Button
+          icon="table"
+          iconOnly
+          text
+          styles={{ color: 'black' }}
```
Comment (sachingupta) [inline src/components/RadioIcons.tsx]: same here.

File: src/components/SearchBar.tsx [hunk 8]
```diff
@@ -37,26 +36,29 @@ export const SearchBar: React.FC<ISearchBarProps> = (props: ISearchBarProps): JS
   };
 
   return (
-    <div className="SearchBar">
-      <RadioIcons onChange={handleRadioButtonChange} />
+    <Flex gap="gap.small" vAlign="center">
+      <Image
```
Comment (sachingupta) [inline src/components/SearchBar.tsx]: use width and height of image object. remove src

**Pull request merged**